wzdx/monitoring/metrics.py

```diff
@@ -0,0 +1,22 @@
+from google.api import label_pb2 as ga_label
```

Comment:
Where is this file referenced/how is it used?

Reply:
Resolved in PR #51